### PR TITLE
[+] Import Raw Private Key (HEX).

### PR DESCRIPTION
**Pull request by Mleekko**
## Add support for importing Raw Private Keys (HEX).

Both new and legacy curves are supported.

## Screenshot

image

## Steps to Verify

Wallet -> add wallet -> "I already have a wallet" -> "Restore from raw private key hex". Then add a new account.

### CODE REVIEW

File: apps/extension/src/pages/keystore/options/index.tsx [hunk 1]
```diff
@@ -34,6 +34,10 @@ const messages = defineMessages({
 		id: '4Q4DB2',
 		defaultMessage: 'Restore from extended private key',
 	},
+	raw_key: {
+		id: '4Q4DB3',
```
Comment (Mleekko) [inline apps/extension/src/pages/keystore/options/index.tsx:38]: Can't figure out the tool that you use for handling i18n strings, so I've simply added these `id`-s randomly. 
Also I didn't add anything to the language bundles - please let me know if I need to fix that (but I will need help!).

Comment (heathsnee) [reply apps/extension/src/pages/keystore/options/index.tsx:38]: eslint should generate this ids for you, its ok i will fix translations before release
usually i do `yarn intl:extract` and then and then run this through ai to generate keys for other languages

**Comment by heathsnee** [comment]
Looks alright, one question before I merge this though:

Is a single hex representation valid for both curves ?
```
				case CURVE.SECP256K1:
					return new PrivateKey.Secp256k1(secret)
				case CURVE.CURVE25519:
					return new PrivateKey.Ed25519(secret)
```

**Comment by Mleekko** [comment]
> Looks alright, one question before I merge this though:
> 
> Is a single hex representation valid for both curves ?
> 
> ```
> 				case CURVE.SECP256K1:
> 					return new PrivateKey.Secp256k1(secret)
> 				case CURVE.CURVE25519:
> 					return new PrivateKey.Ed25519(secret)
> ```
@heathsnee yes, the Private Key is just 32 bytes represented as a 64 char long HEX string. And I also tested both cases.

**Pull request merged**